packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -93,58 +94,48 @@ export class Relayer {
           continue;
         }
         try {
-          await this._relayPendingDeposit(l1Token, relayableDeposit);
+          relayTransactions.push(await this._generateRelayTransactionForPendingDeposit(l1Token, relayableDeposit));
```

Comment:
we simply build an array of relay transactions and then send them all at once using the `processTransactionBatch` Method.

This is the main change in this PR: previously this method would actually run the transactions. now, it simply generates the transaction objects and passes them to the `processTransactionBatch` method to deal with them.